src/main/java/jenkins/plugins/zulip/ZulipNotifier.java

```diff
@@ -174,16 +280,16 @@ private static boolean shouldPublish(Run<?, ?> build) {
      * @param prefix       The prefix to add to the parameter (will be formatted as bold)
      * @param display      The build parameter
      * @param url          The Url to the Jenkins item
-     * @param globalConfig Global step config
      * @return Formatted parameter
      */
-    private static String hundsonUrlMesssage(String prefix, String display, String url, DescriptorImpl globalConfig) {
+    private static String hundsonUrlMesssage(String prefix, String display, String url) {
```

Comment:
Changes that are not part of the smart notification feature should be in a separate commit (or PR).

Never the less, i don't think you should hijack function that was designed to link to Jenkins project / build pages to generate mailto link. I think this should be reverted.

Reply:
In general I didn't break this code. I extended it a bit but the result was the same. So, links to the project / build pages were still generated.

However, I decided to revert this code. My main reason for changing it was overly long lines with the committer info. If author has a long name/surname then in some setups it will be duplicated in the messages sent to the Zulip server. Consider this message:

```
* Johnathan Kristoffersen <johnathan.kristoffersen@example.com> Add test
```

So, I thought that `johnathan.kristoffersen@example.com` can be hidden behind an URL.